### PR TITLE
Update Outpost install process

### PR DESCRIPTION
**Pull request by yunjiang29**

### CODE REVIEW

File: ci-operator/step-registry/aws/provision/edge-zone-subnet/aws-provision-edge-zone-subnet-commands.sh [hunk 1]
```diff
@@ -378,6 +396,53 @@ elif [[ ${EDGE_ZONE_TYPES} == "wavelength-zone" ]]; then
...
+
+  outpost_arn=$(jq -r '.OutpostArn' ${CLUSTER_PROFILE_DIR}/aws_outpost_info.json)
+  outpost_az=$(jq -r '.AvailabilityZone' ${CLUSTER_PROFILE_DIR}/aws_outpost_info.json)
+  echo $outpost_az > ${SHARED_DIR}/outpost_availability_zone
+
+  private_route_table_id=$(head -n 1 "${SHARED_DIR}/private_route_table_id") # TBD
+  public_route_table_id=$(head -n 1 "${SHARED_DIR}/public_route_table_id")
+
+  priv_subnet_cidr="10.0.129.0/24"
+  pub_subnet_cidr="10.0.128.0/24"
```
Comment (jianlinliu) [inline ci-operator/step-registry/aws/provision/edge-zone-subnet/aws-provision-edge-zone-subnet-commands.sh:415]: Does the cidr is bound to the specific outpost ?

Comment (yunjiang29) [reply ci-operator/step-registry/aws/provision/edge-zone-subnet/aws-provision-edge-zone-subnet-commands.sh:415]: No, it's part of VPC CIDR, same as WLZ and LZ

**Comment by yunjiang29** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-outpost-day2-f14

**Comment by yunjiang29** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-aws-ipi-wavelength-byo-subnet-ovn-day2-f28

**Comment by yunjiang29** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-outpost-day2-f14

**Comment by yunjiang29** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-aws-ipi-wavelength-byo-subnet-ovn-day2-f28

**Pull request merged**